tools/nl/embeddings/data/finetuning/sentence_pairs.csv

```diff
@@ -0,0 +1,30 @@
+sentence_1,sentence_2,score
+african americans,black people,1.0
+african americans,black americans,1.0
+african americans,african americans living in america,1.0
+african americans,black people in the US,1.0
+african americans,black and african american demographic,1.0
+african americans,population of black or african americans,1.0
+african americans,foreign born,0.1
+african americans,immigrants,0.1
+african americans,africa,0.2
+african americans,living in africa,0.2
+african americans,poor,0.01
```

Comment:
Maybe we can document these low score threshold choices in the README?  If folks are going to curate this, it will provide some guidance.

Reply:
Done